<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlDock</class>
 <widget class="QDockWidget" name="ControlDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control</string>
  </property>
  <widget class="QWidget" name="ControlDockContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="controlLayout">
      <item>
       <layout class="QHBoxLayout" name="laserControlLayout">
        <item>
         <widget class="QLabel" name="laserStatusLabel">
          <property name="text">
           <string>Laser status:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>laserStatusButton</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="laserStatusButton">
          <property name="text">
           <string>Off</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="photomultiplierControlLayout">
        <item>
         <widget class="QLabel" name="photomultiplierStatusLabel">
          <property name="text">
           <string>Photomultiplier status:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>photomultiplierStatusButton</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="photomultiplierStatusButton">
          <property name="text">
           <string>Off</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="photonReadoutLayout">
        <item>
         <widget class="QLabel" name="photonReadoutTitleLabel">
          <property name="text">
           <string>Current photon count:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="photonReadout10Label">
          <property name="text">
           <string>10s avg.: 1012/s</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="photonReadout1Label">
          <property name="text">
           <string>1s avg.: 992/s</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="photonReadout01Label">
          <property name="text">
           <string>0.1s avg.: 1152/s</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="accelerationControlLayout">
        <item>
         <widget class="QLabel" name="accelerationLabel">
          <property name="text">
           <string>Acceleration voltage: 1200V</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="accelerationSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="triggerControlLayout">
        <item>
         <widget class="QLabel" name="triggerLabel">
          <property name="text">
           <string>Trigger voltage: 1200V</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="triggerSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="positionControlLayout">
        <item>
         <layout class="QHBoxLayout" name="positionStatusLayout">
          <item>
           <widget class="QLabel" name="currentPositionLabel">
            <property name="text">
             <string>Current position: 10.02mm</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="targetPositionLabel">
            <property name="text">
             <string>Target position: 11.33mm</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSlider" name="positionSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="positionControl1Layout">
          <item>
           <widget class="QPushButton" name="positionFineLeftButton">
            <property name="text">
             <string>&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="positionLineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="positionFineRightButton">
            <property name="text">
             <string>&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="positionControl2Layout">
          <item>
           <widget class="QPushButton" name="positionCoarseLeftButton">
            <property name="text">
             <string>&lt;&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="positionHomeButton">
            <property name="text">
             <string>Home</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="positionCoarseRightButton">
            <property name="text">
             <string>&gt;&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
